great_expectations/render/types/__init__.py

```diff
@@ -510,9 +510,9 @@ def __init__(
         graph: Optional[dict] = None,
     ):
         # StringValueType
-        self.template: str = template
-        self.params: dict = params
-        self.schema: dict = schema
+        self.template: Optional[str] = template
+        self.params: Optional[dict] = params
+        self.schema: Optional[dict] = schema
```

Comment:
Wai ...whu? Adding Optional?

Reply:
weirdly enough yes! if you look at the args being passed into the constructor, they're all `Optional`. The attr assignment should reflect that right?